META.json

```diff
@@ -0,0 +1,37 @@
+{
+   "name": "mvtbl",
+   "abstract": "Easily move tables around tablespaces",
+   "version": "0.0.2",
+   "license": {
+      "PostgreSQL": "http://www.postgresql.org/about/licence"
+   },
+   "provides": { 
+      "mvtbl": {
+         "file": "sql/mvtbl.sql",
+         "version": "0.0.2",
+         "abstract": "Easily move tables around tablespaces"
+      }
+   },
+   "meta-spec": {
+     "version": "0.0.2",
+     "url": "http://pgxn.org/meta/spec.txt"
+   },
+   "description": "Easily move tables around tablespaces",
+   "prereqs": {
+      "runtime": {
+        "requires": {
+           "PostgreSQL": "9.3.0"
```

Comment:
We test only PostgreSQL 10+ so it makes sense to change it to `"10.0"`:
https://github.com/adjust/pg-mvtbl/blob/master/.github/workflows/ci.yml#L14